internal/olm/operator/registry/index/registry_pod.go

```diff
@@ -50,6 +50,8 @@ type BundleItem struct {
 	ImageTag string `json:"imageTag"`
 	// AddMode describes how the bundle should be added to an index image.
 	AddMode BundleAddMode `json:"mode"`
+	// SkipTLS controls wether to ignore SSL errors while pulling bundle image from registry server.
+	SkipTLS bool `json:"SkipTLS"`
```

Comment:
This field should be added to `RegistryPod` not `BundleItem` since it's globally configured and not on a per-bundle basis.

Reply:
that's also where the preexisting cert stuff is anyway :+1:

Reply:
agreed

Reply:
Done. Changed as @estroz suggested.